libmscore/cmd.cpp

```diff
@@ -1999,6 +1999,18 @@ bool Score::processMidiInput()
                         ev.chord = false;
                   else
                         ev.chord = true;
+
+                  Element* cr = _is.lastSegment()->element(_is.track());
+                  if (cr->isChord()) {
+                        Note* n = toChord(cr)->findNote(ev.pitch);
+                        if (n) {
+                              deleteItem(n->tieBack());
+                              deleteItem(n);
```

Comment:
this line deleteItem(n) is what causes problems with https://musescore.org/en/node/280715 as far as I can tell.  Removing it fixes that problem, while still allowing me to hold shift and enter notes with piano to form chord.  I'm trying to figure out the reason why you would delete the actual previous note...

Reply:
maybe you meant to delete something else here related to the note?  But I can't seem to figure out a reason why delete the note.  I'm submitting a PR to remove this line...

Reply:
I see...I believe you wanted to prevent the situation of having multiple noteheads for identical pitches appear in the same chord.  In that case, this deleteItem(n) should only happen if that n is in the same chordRest as being inputted currently (which happens when shift is held down, but is not the case when shift is not held down).

Reply:
I'm only going to delete that note if ev.chord is true.  I think that is the intended behavior.

Reply:
I think I figured it out...PR emminent.

Reply:
and here is my PR: #4510

@jthistle might want to review: